src/test/java/resources/voucherManagementTest.feature

```diff
@@ -77,6 +79,23 @@ Feature: Voucher Management Api Test
     Given I can create an RedeemVoucherRequestDTO for voucher redemption
     When I call the redeem voucher API with expected status of 200
 
+ @gov
+  Scenario: Unsupported Parameter Validation for Create Voucher API test
+    Given I can create an VoucherRequestDTO for voucher creation with unsupported parameter parameter
+    When I call the create voucher API with expected status of 400 and stub "/createVoucher"
+    Then I should be able to assert the create voucher validation for negative response
+
+#  @gov @createAndActivateVoucher
+#  Scenario: Redeem Voucher Api Test
+#    Given I can create an RedeemVoucherRequestDTO for voucher redemption
+#    When I call the redeem voucher API with expected status of 200
+#    Then I can assert that redemption was successful by asserting the status in response
+
+  @gov @createAndActivateVoucher
+   Scenario: Conflicting/unique data validations check for Create Voucher API
+    Then I will call the fetch voucher API with expected status of 200
+    When I call the create voucher API with expected status of 409 and stub "/createVoucher"
```

Comment:
how the duplicate is assured in this test?

Reply:
We are first creating and activating the voucher using @createAndActivateVoucher. The functions itself will generate the unique value for DTO.
And also insuring the voucher is created by fetching the voucher. Then, we are again trying to create the voucher with same DTO(having same/conflicting field). So, it will give "409" as status.